src/js/utils/browser.js

```diff
@@ -48,10 +51,10 @@ export const ANDROID_VERSION = (function() {
     return parseFloat(match[1] + '.' + match[2]);
   } else if (major) {
     return major;
-  } else {
-    return null;
   }
-})();
+  return null;
+}());
```

Comment:
I plead the 5th.
(Actually, I don't remember setting that one... might be better to change it so we have less changes but I'm not attached to this.)